### ISSUE TITLE
Unable to Parse QueryExecute Functions

### ISSUE THREAD

**Issue by jerronjames** [cflint/CFLint#49]
```
                var result = QueryExecute("
			SELECT info_card_id, document_num, revision_nm, title_nm
			FROM tdc_doc_infocard
			WHERE info_card_id = :sourceTemplateNum
			ORDER BY UPPER(document_num) ASC, revision_seq DESC
		", { sourceTemplateNum = { value="#sourceTemplateNum#", cfsqltype="cf_sql_varchar" }});
```
Is reporting:

decompile: [ public function getSourceTemplates(required sourceTemplateNum) {, var result = QueryExecute(' , SELECT info_card_id, document_num, revision_nm, title_nm , FROM tdc_doc_infocard , WHERE info_card_id = :sourceTemplateNum , ORDER BY UPPER(document_num) ASC, revision_seq DESC , ', {sourceTemplateNum:{value:'#sourceTemplateNum#',cfsqltype:'cf_sql_varchar'}}); return result;, }] { var result = QueryExecute(' SELECT info_card_id, document_num, revision_nm, title_nm FROM tdc_doc_infocard WHERE info_card_id = :sourceTemplateNum ORDER BY UPPER(document_num) ASC, revision_seq DESC ', {sourceTemplateNum:{value:'#sourceTemplateNum#',cfsqltype:'cf_sql_varchar'}}); return result; }

**Comment by ryaneberly** [comment]
the parser seems to get it ok.  

Is it an issue with the CFLint objects and the decompile method?

**Comment by ryaneberly** [comment]
@jerronjames, That's from the FunctionLengthChecker isn't it?
it's printing a decompile of the function body twice.

**Comment by jerronjames** [comment]
Indeed it is with the function length checker. I just excluded it, and it ran fine. Guess I’ll be working on that next week!
